packages/strapi-plugin-upload/admin/src/components/UploadList/index.js

```diff
@@ -23,32 +22,34 @@ const UploadList = ({
     filesToUploadLength > 1 ? 'plural' : 'singular'
   }`;
 
-  // TODO: use <IntlText ... /> instead of FormattedMessage
-
   return (
     <>
       <ModalSection justifyContent="space-between">
         <TextWrapper>
-          <Text fontSize="md" fontWeight="bold">
-            <FormattedMessage id={getTrad(titleId)} values={{ number: filesToUploadLength }} />
-          </Text>
-
-          <Text fontSize="sm" color="grey">
-            <FormattedMessage
-              id={getTrad('modal.upload-list.sub-header-subtitle')}
-              values={{ number: filesToUploadLength }}
-            />
-          </Text>
+          <IntlText
+            id={getTrad(titleId)}
+            values={{ number: filesToUploadLength }}
+            fontSize="md"
+            fontWeight="bold"
+            lineHeight="19px"
```

Comment:
Why is the `line-height` set to `19px` when our baseline is `18px` ? I believe that you should find another way.